tools/packaging/release/update-repository-version.sh

```diff
@@ -246,7 +246,7 @@ EOT
 		${hub_bin} push fork -f "${branch}"
 		info "Create PR"
 		out=""
-		out=$("${hub_bin}" pull-request -b "${target_branch}" -F "${notes_file}" 2>&1) || echo "$out" | grep "A pull request already exists"
+		out=$("${hub_bin}" pull-request -b "${target_branch}" -F "${notes_file}" 2>&1) || echo "$out" | LC_ALL=C LANG=C grep "A pull request already exists"
```

Comment:
Shouldn't `LC_ALL=C LANG=C` be in the environment of `hub_bin` process instead? IIUC it is `hub_bin` which can change the output message based on the locale...

Reply:
Yep, good catch!